vadl/main/vadl/ast/vadl.ATG

```diff
@@ -860,6 +871,43 @@ PRODUCTIONS
                                                 } .)
   .
 
+  clangTypeDefinition<out Definition def>                     (. def = DUMMY_DEF; .)
+  =
+  (
+    SIZE TYPE SYM_EQ typeSizeDefinition<out AbiClangTypeDefinition.TypeSize size1> (. def = new AbiClangTypeDefinition(loc(), AbiClangTypeDefinition.TypeName.SIZE_TYPE, size1); .)
+    | INT MAX TYPE SYM_EQ typeSizeDefinition<out AbiClangTypeDefinition.TypeSize size2> (. def = new AbiClangTypeDefinition(loc(), AbiClangTypeDefinition.TypeName.INT_MAX_TYPE, size2); .)
+  )
+  .
+
+  clangNumericTypeDefinition<out Definition def>                     (. def = DUMMY_DEF; .)
+  =
+  (
+    POINTER (
+      WIDTH SYM_EQ expression<out Expr expr1, BIN_OPS>               (. def = new AbiClangNumericTypeDefinition(loc(), AbiClangNumericTypeDefinition.TypeName.POINTER_WIDTH, expr1); .)
+      |  ALIGN SYM_EQ expression<out Expr expr2, BIN_OPS>            (. def = new AbiClangNumericTypeDefinition(loc(), AbiClangNumericTypeDefinition.TypeName.POINTER_ALIGN, expr2); .)
+    )
+    | LONG (
+      WIDTH SYM_EQ expression<out Expr expr3, BIN_OPS>               (. def = new AbiClangNumericTypeDefinition(loc(), AbiClangNumericTypeDefinition.TypeName.LONG_WIDTH, expr3); .)
+      |  ALIGN SYM_EQ expression<out Expr expr4, BIN_OPS>            (. def = new AbiClangNumericTypeDefinition(loc(), AbiClangNumericTypeDefinition.TypeName.LONG_ALIGN, expr4); .)
+    )
+  )
```

Comment:
@flofriday I am open to suggestions if you have any to make it more concise haha

Reply:
I don't see anything at the moment and we can still refactor it when we discover something 👍😉